src/reports/budgetingperf.cpp

```diff
@@ -205,7 +205,7 @@ wxString mmReportBudgetingPerformance::getHTMLText()
                         wxString estimateVal = Model_Currency::toString(estimate, Model_Currency::GetBaseCurrency());
                         //make the href string parameters
                         std::stringstream ss;
-                        ss << "budget: " << estimateVal << "|" << Model_Currency::toString(actual, Model_Currency::GetBaseCurrency()) << "|" << catID << "|"
+                        ss << "budget:" << estimate << "|" << actual << "|" << catID << "|"
```

Comment:
@whalley  Thank you for fixing budget link. I noticed that decimal on the dialog doesn't work fine after your commit:
![image](https://github.com/user-attachments/assets/3a1578dc-5d47-4a7e-b41d-239ec1d270f6)
could be caused by changing estimate and actual in the passed string?

Reply:
Ooops. Does it work fine if you just revert line 208?